src/nyc_trees/js/src/map.js

```diff
@@ -188,28 +201,10 @@ function addGridLayer(map, options) {
             crosshairs: options.crosshairs || false,
             pointerCursor: !options.crosshairs
         });
-    _addLayer(map, layer, options.waitForZoom);
+    map.addLayer(layer);
     return layer;
 }
 
-function _addLayer(map, layer, waitForZoom) {
-    if (waitForZoom) {
-        _addAfterZoom(map, layer);
-    } else {
-        map.addLayer(layer);
-    }
-}
-
-function _addAfterZoom(map, layer) {
-    // Add layer to map after zoom animation completes.
-    // (Otherwise spurious tile requests will be issued at the old zoom level.)
```

Comment:
Consider restoring this comment to https://github.com/azavea/nyc-trees/pull/1604/files#diff-2780080b0b4bcd298f4a429c851f2f8bR128